wingtips-lightstep/src/main/java/com/nike/wingtips/lightstep/WingtipsToLightStepLifecycleListener.java

```diff
@@ -98,8 +98,17 @@ public void spanStarted(Span wingtipsSpan) {
         // Do nothing
     }
 
+    protected boolean shouldReportCompletedSpan(Span span) {
```

Comment:
hmm I smell a 100% sampling thing somewhere :D

Reply:
Hah! You're not wrong... ;)